tools/setup_helpers/generate_code.py

```diff
@@ -79,6 +77,56 @@ def generate_code(ninja_global=None,
             python_install_dir,
             autograd_dir)
 
+def get_selector_from_legacy_operator_selection_list(
+        selected_op_list_path: str,
+):
+    from tools.autograd.utils import load_op_list_and_strip_overload
+
+    selected_op_list = load_op_list_and_strip_overload(
+        None,
+        selected_op_list_path,
+    )
+
+    # Internal build doesn't use this flag any more. Only used by OSS
+    # build now. Every operator should be considered a root operator
+    # (hence generating unboxing code for it, which is consistent with
+    # the current behaviour), and also be considered as used for
+    # training, since OSS doesn't support training on mobile for now.
+    #
+    is_root_operator = True
+    is_used_for_training = True
+
+    from tools.codegen.selective_build.selector import SelectiveBuilder
+
+    selector: SelectiveBuilder = SelectiveBuilder.get_nop_selector()
+    if selected_op_list is not None:
+        selector = SelectiveBuilder.from_legacy_op_registration_allow_list(
+            selected_op_list,
+            is_root_operator,
+            is_used_for_training,
+        )
+
+    return selector
+
+
+def get_selector(selected_op_list_path, operators_yaml_path):
+    # cwrap depends on pyyaml, so we can't import it earlier
+    root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    sys.path.insert(0, root)
```

Comment:
@ljk53 I ran into a problem in a windows CI build (iirc) because the paths aren't set up correctly. In fact the comment above about `cwrap` doesn't make sense to me since it's not used in this file at all. I agree that this should be put into main. Let me send out another change set on top of this one to address it and clean it up.